modules.d/99openssl/module-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+check() {
+    return 255
+}
+
+install() {
+    local ossl_files="${dracutbasedir}/ossl-files"
+
+    local openssl_cnf="$($ossl_files --config)"
+
+    local initrd_openssl_cnf="${initdir}/${openssl_cnf}"
+
+    if [[ ! -r "$openssl_cnf" ]]; then
+        dfatal "'$ossl_files --config' does not return a path!!"
+        exit 1
+    fi
+
+    inst_multiple -o \
+      /etc/pki/tls/openssl.cnf \
+      /etc/pki/tls/openssl.d/* \
```

Comment:
These can be removed, the invocation of `ossl-config` makes sure that it exists and does not use any includes.

Reply:
Thanks; I wasn't sure about that. Shloudn't I keep it just for redundancy though?

Reply:
I wouldn't. A modified `/etc/pki/tls/openssl.cnf` could contain `.include /opt/foobar/module.cnf`, which we wouldn't copy, and could case an issue. `.include /etc/pki/tls/openssl.d/` isn't the only include statement that could be in this file.

For that reason, I'd also remove the `openssl.d` directory. It shouldn't be needed anymore.